services/IoT/applications/libraries/java.md

```diff
@@ -207,7 +207,7 @@ The following code provides a sample implementation of the Event callback:
     }
 ```
 
-When the event callback is added to the ApplicationClient, the processEvent() method is invoked whenever any event is published on the subscribed criteria, The following snippet shows how to add the Event call back into ApplicationClient instance,
+When the event callback is added to the ApplicationClient, the processEvent() method is invoked whenever any event is published on the subscribed criteria. The following snippet shows how to add the event callback into ApplicationClient instance:
```

Comment:
The phrase "whenever any event is published on the subscribed criteria" is not clear. Do you mean "whenever an event is publish for the subscribed criteria" or "whenever an event is published that contains the subscribed 
criteria" ? Clarify.